backend/core/src/listings/listings.controller.ts

```diff
@@ -44,7 +44,13 @@ import { defaultValidationPipeOptions } from "../shared/default-validation-pipe-
 @UseGuards(OptionalAuthGuard, AuthzGuard)
 @UsePipes(new ValidationPipe(defaultValidationPipeOptions))
 export class ListingsController {
-  constructor(private readonly listingsService: ListingsService) {}
+  cacheKeys: string[]
+  constructor(
+    @Inject(CACHE_MANAGER) private cacheManager,
+    private readonly listingsService: ListingsService
+  ) {
+    this.cacheKeys = ["/listings", "/listings?filter[$comparison]=%3C%3E&filter[status]=pending"]
```

Comment:
What is the meaning of this expression?

Reply:
The cache entry keys are the route path by default, so the first one is from partners and the second is coming from public. We can also specify the cache key with a decorator per operation, if we don't want it based off the route.

Reply:
Also, rather than setting the keys like this, we could get the keys via `cacheManager.keys()` and filter out the keys for individual listings (i.e. /listings/[id]), so we only modify the "list" entries.